.env.test

```
@@ -7,4 +7,4 @@ PYTEST_ADMIN_PASSWORD=start123
 PYTEST_DEFAULT_MASTER_IMAGE=python/base
 PYTEST_ASYNC_MAX_RETRIES=5
 PYTEST_ASYNC_RETRY_DELAY_MILLIS=500
-PYTEST_HUB_VERSION=0.8.21
+PYTEST_HUB_VERSION=0.8.25
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PYTEST_HUB_VERSION key should go before the PYTEST_STORAGE_BASE_URL key

(UnorderedKey)